### PR TITLE
Fe config sync

### PR DESCRIPTION
**Pull request by lumburovskalina**
Main PR: https://github.com/code-kern-ai/cognition-ui/pull/67

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,10 +37,11 @@
     "react-chartjs-2": "^5.2.0",
     "react-dom": "18.2.0",
     "react-dropzone": "^14.2.3",
-    "react-redux": "^8.1.1",
+    "react-i18next": "^14.1.2",
```
Comment (LennartSchmidtKern) [inline package.json:40]: Where are these package requirements now coming from? i18next is still only used in cognition-ui.

Comment (LennartSchmidtKern) [reply package.json:40]: Got explanations in person: related to submodules shared across

File: src/components/projects/projectId/heuristics/heuristicId/shared/HeuristicRunButtons.tsx [hunk 2]
```diff
@@ -21,6 +21,7 @@ export default function HeuristicRunButtons(props: HeuristicRunButtonsProps) {
 
     useEffect(() => {
         setCanStartHeuristic(checkCanStartHeuristic());
+        if (!props.checkCanStartHeuristic) return;
```
Comment (LennartSchmidtKern) [inline src/components/projects/projectId/heuristics/heuristicId/shared/HeuristicRunButtons.tsx:24]: This is some fix?

Comment (LennartSchmidtKern) [reply src/components/projects/projectId/heuristics/heuristicId/shared/HeuristicRunButtons.tsx:24]: If so, shouldnt it be in dep array also?

Comment (lumburovskalina) [reply src/components/projects/projectId/heuristics/heuristicId/shared/HeuristicRunButtons.tsx:24]: yes i noticed one bug while dev, so i fixed it, i will add it as part of the dep array

**Review by LennartSchmidtKern — APPROVED** [review]

**Pull request merged**